lib/sassc/engine.rb

```diff
@@ -17,6 +17,8 @@ def initialize(template, options = {})
     end
 
     def render
+      return @template if @template.empty?
+
```

Comment:
Wouldn't it be better to replace it with `return '' if @template.empty?`? It may seem the same but it isn't. With the current implementation we would get a reference to the original source and maybe change it after calling `render` causing a side effect on the original source which could cause some unexpected behaviors if one assume `render` would always return a new string...